assets/src/adaptivity/rules-engine.ts

```diff
@@ -1,3 +1,4 @@
+import { JanusConditionProperties } from 'apps/authoring/components/AdaptivityEditor/ConditionsBlockEditor';
```

Comment:
I think we need to split this type out into a different file. the rules engine is part of the server build for the engine and should not depend on a file inside the app. we can probably reverse it and have the app depend on a type in the rules-engine

Reply:
Done